guide/designing-products/user-research.md

```diff
@@ -57,19 +57,18 @@ We design digital products for the benefit of others. To do that, we need to und
    layout = "float-left"
 %}
 
-**In short, user research helps teams create better products in a more focused and efficient manner.**
+**User research helps teams efficiently create better and more focused products**
 
-Prioritizing on the features and improvements that make the biggest difference is crucial. What precisely those are is often a matter of heavy debate with clashing opinions and rationale. A great way to get past individual views is to make data-informed decisions.
+Prioritizing features that make the most significant difference is crucial. Those features are seldom apparent, and their discovery is often the result of debate and critical thinking. If you find yourself stuck, try making data-informed decisions.
 
-User research creates data from the start and end points of our efforts. Products are created because there are specific unaddressed needs, then they go through development and finally back in the hands of the target audience for use and evaluation. We are often too occupied with the complexities of development, but the real measure of success is whether users' needs are satisfied.
+The results are appropriate product roadmaps, better-focused team efforts, and happier users. This is especially important for projects with limited resources, in which many open-source efforts are.
```

Comment:
Removes a point I was trying to explicitly make about being too focused on the tech while not really addressing user needs. It was meant to make the reader question whether this is something they are "guilty of".

Reply:
Reverted in https://github.com/BitcoinDesign/Guide/pull/365/commits/7a005dfb2b53e00b399f33ff33963a3f2a505a3d